extensions/wikia/Recirculation/RecirculationContent.php

```diff
@@ -27,23 +29,25 @@ class RecirculationContent implements JsonSerializable {
 	 * @param $publishDate
 	 * @param $author
 	 */
-	public function __construct($index, $url, $thumbnail, $title, $publishDate, $author) {
-		$this->index = $index;
-		$this->url = $url;
-		$this->thumbnail = $thumbnail;
-		$this->title = $title;
-		$this->publishDate = $publishDate;
-		$this->author = $author;
+	public function __construct( array $properties = [] ) {
+		foreach($properties as $key => $value){
+			$this->{$key} = $value;
+		}
+	}
+
+	public function get( $property ){
+		return isset( $this->{ $property } ) ? $this->{ $property } : '';
 	}
 
 	function jsonSerialize() {
```

Comment:
Added public, but the name is based on http://php.net/manual/en/jsonserializable.jsonserialize.php